pep-0619.rst

```diff
@@ -83,7 +83,12 @@ Features for 3.10
 
 Some of the notable features of Python 3.10 include:
 
+* :pep:`604`, Allow writing union types as ``X | Y``
+* :pep:`612`, Parameter Specification Variables
+* :pep:`613`, Explicit Type Aliases
 * :pep:`618`, Add Optional Length-Checking To ``zip``
+* :pep:`626`, Precise line numbers for debugging and other tools
+* :pep:`634`, :pep:`635`, :pep:`636`, Structural Pattern Matching
```

Comment:
We are missing https://www.python.org/dev/peps/pep-0644/, could you add it to the list @east825 ?

Reply:
@pablogsal Sure thing! I didn't notice it because it wasn't even mentioned in What's New. BTW, is there a definitive way to track which PEPs are accepted and planned for a particular release? Currently, I gather this information from Twitter announcements, newsletters, What's New, etc., but this approach is not flawless :)

Reply:
> BTW, is there a definitive way to track which PEPs are accepted and planned for a particular release?

No other than the official release announcements such as https://www.python.org/downloads/release/python-3100a6/

Reply:
Aha, that will do, thanks!